zebrad/src/components/sync.rs

```diff
@@ -32,20 +38,21 @@ const FANOUT: usize = checkpoint::MAX_QUEUED_BLOCKS_PER_HEIGHT;
 ///
 /// When we implement a peer reputation system, we can reduce the number of
 /// retries, because we will be more likely to choose a good peer.
-const BLOCK_DOWNLOAD_RETRY_LIMIT: usize = 5;
+const BLOCK_DOWNLOAD_RETRY_LIMIT: usize = 2;
```

Comment:
It would be nice to get rid of the outer `Retry`, but I think that would mean adding a retry limit within the `Hedge`.

And maybe also an override for skipping retries when there are too few data points in the histogram.